lock/src/main/java/com/auth0/android/lock/views/ValidatedInputView.java

```diff
@@ -115,6 +117,14 @@ private void setupInputValidation() {
                 inputErrorIcon = R.drawable.com_auth0_lock_ic_input_username_error;
                 hint = getResources().getString(R.string.com_auth0_lock_hint_username);
                 break;
+            case CODE:
+                input.setInputType(InputType.TYPE_NUMBER_VARIATION_NORMAL);
+                //Fixme: Code length is configured in the dashboard. How can we get this value?
```

Comment:
We should not restrict a limit of characters since we don't know the preferred length